src/Locales/lv_LV.php

```diff
@@ -33,7 +33,7 @@
     'relativeTime'  => array(
         'future' => 'pēc %s',
         'past'   => 'pirms %s',
-        's'      => 'sekundes',
+        's'      => 'dažas sekundes',
```

Comment:
Better will be: `   's'      => 'dažām sekundēm', `

Reply:
Done. Thanks @LinMAD.